h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -397,6 +397,13 @@ public final boolean hasClosedForm(long na_cnt) {
         InitialXSVD xtsk = new InitialXSVD(dsqrt, _parms._k, _ncolA, _ncolX);
         xtsk.doAll(fullFrm);
 
+        // fix for PUBDEV-3528.  Copy content of fullFrm into dfrm for X, W
```

Comment:
Tomas:

The first assignment is necessary for large datasets like the one from the customer.  If you do not copy the matrix over, the X matrix will be initialized with zero when the function return.  If you have the dataset, you can run the pyunit_glrm_PUBDEV_3454_NOFEATURE.py.  If you do not have the dataset, I will grab you tomorrow and show it to you.  Thanks!